pkg/controllers/management/oidcprovider/controller.go

```diff
@@ -102,10 +92,13 @@ func (c *oidcClientController) onChange(_ string, oidcClient *v3.OIDCClient) (*v
 		if err != nil {
 			return nil, fmt.Errorf("failed to generate client secret: %w", err)
 		}
-
+		clientSecretName := secretKeyPrefix + "1"
 		k8sSecret, err = c.secretClient.Create(&v1.Secret{
 			ObjectMeta: metav1.ObjectMeta{
-				Name:      clientID,
+				Name: clientID,
+				Annotations: map[string]string{
+					clientSecretCreatedAtPrefixAnn + clientSecretName: fmt.Sprintf("%d", c.now().Unix()),
```

Comment:
Probably not gonna make much of a difference, but this use of `fmt.Sprintf` is likely to be around 3x slower than using `strconv.FormatInt`.

Reply:
good point, but that's not needed in this case